pkgs/applications/editors/vscode/update.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+command -v grep >/dev/null 2>&1 || { echo >&2 "[grep] is required, but not installed.  Aborting."; exit 1; }
```

Comment:
Deps can be included by using `nix-shell`. 

Example: https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/minecraft/update.sh

Reply:
Fixed in #90224